### DISCUSSION TITLE
How to get chat messages

### DISCUSSION THREAD

**Discussion by strangerage** [akiver/cs-demo-manager#731] [Q&A] [answered]
Hi, 

May I know how may I get chat messages/chat logs from a replay? When i tried to press the chat tab, it said "No chat messages found. Chat messages are available only if the .info file of the demo is next to its .dem file during analysis.

Thanks

**Comment by akiver** [comment, marked as answer]
As far as I know, chat messages are currently unavailable in CS2 demos.
The v3.2.0 removes the tab only for CS2 demos - the day chat messages are available in CS2 demos I will add it back.

**Reply by akiver** [reply]
For information, chat messages have been available since v3.8.0 but Valve MM demos still don't include them.
You can extract chat messages from demos coming from third-party services such as Faceit.
